packages/client/src/components/BigMovieCard/BigMovieCard.jsx

```diff
@@ -49,7 +49,7 @@ const BigMovieCard = () => {
             border: 1,
             borderColor: 'grey.500',
           }}
-          src="https://www.themoviedb.org/t/p/w300_and_h450_bestv2/9xjZS2rlVxm8SFx8kPC3aIGCOYQ.jpg"
+          src={currentMovie?.image_location}
```

Comment:
To avoid empty page refactored the component to use fallBackMovies.json as a default state. We will also provide a message to user about the error with ID, but on a later stage, after we have arranged designs of a page. (it could be something like "you are trying to access non-existing movie: showing default choice in stead") But we really need to focus on styles right now.